h/services/bulk_annotation.py

```diff
@@ -110,10 +113,17 @@ def annotation_search(
     def _search_query(cls, authority, audience, created) -> Select:
         """Generate a query which can then be executed to find annotations."""
         return (
-            sa.select([cls._AUTHOR.username, Group.authority_provided_id])
+            sa.select(
+                [
+                    cls._AUTHOR.username,
+                    Group.authority_provided_id,
+                    sa.func.coalesce(AnnotationMetadata.data, "{}").label("metadata"),
+                ]
+            )
             .select_from(AnnotationSlim)
             .join(cls._AUTHOR, cls._AUTHOR.id == AnnotationSlim.user_id)
             .join(Group, Group.id == AnnotationSlim.group_id)
+            .outerjoin(AnnotationMetadata)
```

Comment:
Using `outerjoin()` because not every `annotation` has an `annotation_metadata` (and we want to return `metadata={}` for the ones that don't)